src/test/passes/arithmetic/report_diff-x86_64-linux.c

```diff
@@ -0,0 +1,89 @@
+// REQUIRES: x86-registered-target
+
+// RUN: env OMVLL_CONFIG=%S/report_diff.py clang -target x86_64-pc-linux-gnu -fpass-plugin=%libOMVLL -O1 -S %s -o /dev/null | FileCheck %s
+
+// CHECK: omvll::Arithmetic applied obfuscation
```

Comment:
FYI This is the output I get with LLVM 18.x, I will check again next week with the current version 14.x and (probably) update this

Reply:
It indeed looks so: https://github.com/weliveindetail/o-mvll/actions/runs/9716209317/job/26819320886.

Reply:
Looks good now: https://github.com/weliveindetail/o-mvll/actions/runs/9744886045/job/26891509692#step:3:14650